api/test/wfl/integration/google/bigquery_test.clj

```diff
@@ -33,12 +17,6 @@
     (is (every? #(= (:kind %) "bigquery#table") tables))))
 
 (deftest test-query-table-sync
-  (let [query-result (bigquery/query-table-sync google-project dr-dataset dr-view)]
+  (let [query (format "SELECT * FROM `%s.%s.%s`" google-project dr-dataset dr-view)
+        query-result (bigquery/query-sync google-project query)]
     (is (= "bigquery#queryResponse" (:kind query-result)))))
```

Comment:
Why are you testing this?

Reply:
Surely you'd want to make sure the data is what you expected, not that you got a big query response?